frontend/src/FormattedText.js

```diff
@@ -2,14 +2,16 @@ import ReactMarkdown from 'react-markdown';
 import remarkUnwrapImages from 'remark-unwrap-images';
 import { remarkDefinitionList, defListHastHandlers } from 'remark-definition-list';
 import CroppedImage from './CroppedImage';
+import Comment from './Comment';
 
 function FormattedText({children}) {
   return (
     <ReactMarkdown
       remarkPlugins={[remarkDefinitionList, remarkUnwrapImages]}
       components={{
         img: (x) => embedVideo(x) || CroppedImage(x),
-        a: ({children, href}) => <a href={href}>{children}</a>
+        a: ({children, href}) => <a href={href}>{children}</a>,
+        p: (x) => Comment(x) || <p>{x.children}</p>
```

Comment:
You may insert `p` before `a`, so that you don't get the `blame` of the line with `a`.

Reply:
Done